charts/client/chart.scss

```diff
@@ -1,7 +1,7 @@
 $sans-serif-font-stack: Lato, "Helvetica Neue", Arial, sans-serif;
 $serif-font-stack: "Playfair Display", Georgia, "Times New Roman", serif;
 
-$controls-color: #3F9EFF;
+$controls-color: #3f9eff;
```

Comment:
Looks like a bunch of unrelated style changes in this file owing to Prettier? Not a big deal but personally I would be inclined to do a separate PR for that. It's a bit of a pain but it keeps unrelated changes apart.

Reply:
:+1: I agree